src/server-extension/resolvers/ChannelsResolver/index.ts

```diff
@@ -500,9 +486,8 @@ export class ChannelsResolver {
         await addNotification(
           em,
           account,
-          new ChannelExcluded({
-            recipient: new ChannelRecipient({ channelTitle: parseChannelTitle(channel) }),
-          })
+          new ChannelRecipient({ channel: channel.id }),
+          new ChannelExcluded({})
```

Comment:
Still https://github.com/Joystream/atlas/issues/4891#issuecomment-1725299795 proposes to make this a member notification because currently this notification isn't accessible once the channel is excluded